docs/sources/shared/visualizations/axis-options-all.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Axis options
+comments: |
+  Most visualizations with axis options use this file. Some, like state timeline and status history, should use axis-options-size as they only have the size option.
```

Comment:
This text should just indicate what files are using this file for the sake of review, rather than where we might/should use this file. Based on my last review of all the visualization settings, there are three visualizations besides state timeline and status history that have axis options (time series, bar chart, xy chart) and all three of them are slightly different so a shared file might not be a great solution for them at the moment. For the sake of not slowing this down, I'll leave it here, but I might opt to remove it later.

```suggestion
  This file is used in the following visualizations: time series.
```